app/reports/service_requests.rb

```diff
@@ -99,6 +100,10 @@ def column_attrs
       end
     end
 
+    if params[:fulfillment_info]
+      attrs["Sent to SPARCFulfillment"] = "service_request.try(:protocol).try(:sub_service_request).try(:in_work_fulfillment?) ? 'Yes' : 'No'"
```

Comment:
Ah, gotcha. I'm backwards here. Do you think it's more appropriate to do?

    service_request.sub_service_requests.in_work_fulfillment.any?